lib/assesments/assessment_2/fi-as2-hira/Screens/Login/fi_as2_hira_login.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi-as2-hira/Screens/Home/fi_as2_hira_home.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi-as2-hira/constants/fi_as2_hira_colors.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi-as2-hira/Screens/Login/widgets/fi_as2_hira_stacklogo.dart';
+import '../../widgets/core/fi_as2_hira_button.dart';
+
+class H1Login extends StatelessWidget {
+  const H1Login({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    Widget smallspace = const SizedBox(
+      height: 35,
+    );
+    return Scaffold(
+      backgroundColor: Colors.white,
+      body: SafeArea(
+        child: Column(
```

Comment:
Use `mainAxisAlignment: MainAxisAlignment.center` and this will adjust the widgets in the center very efficiently